packages/expo-updates/e2e/setup/project.js

```diff
@@ -143,6 +143,35 @@ async function copyCommonFixturesToProject(projectRoot, appJsFileName, repoRoot)
 
   // copy .prettierrc
   await fs.copyFile(path.resolve(repoRoot, '.prettierrc'), path.join(projectRoot, '.prettierrc'));
+
+  // Modify specific files for TV
+  if (isTV) {
+    // Modify .detoxrc.json for TV
+    const detoxRCPath = path.resolve(projectRoot, '.detoxrc.json');
+    let detoxRCText = await fs.readFile(detoxRCPath, { encoding: 'utf-8' });
+    detoxRCText = detoxRCText.replace(/iphonesim/g, 'appletvsim').replace('iPhone 14', 'Apple TV');
+    await fs.rm(detoxRCPath);
+    await fs.writeFile(detoxRCPath, detoxRCText, { encoding: 'utf-8' });
+
+    // Add TV environment variable to EAS build config
+    const easJsonPath = path.resolve(projectRoot, 'eas.json');
+    let easJson = require(easJsonPath);
+    easJson = {
+      ...easJson,
+      build: {
+        ...easJson.build,
+        updates_testing: {
+          ...easJson.build.updates_testing,
+          env: {
+            ...easJson.build.updates_testing.env,
+            TEST_TV_BUILD: '1',
+          },
+        },
+      },
+    };
+    await fs.rm(easJsonPath);
+    await fs.writeFile(easJsonPath, JSON.stringify(easJson, null, 2), { encoding: 'utf-8' });
```

Comment:
don't think you need the `rm` as `writeFile` overwrites by default

Reply:
This is me being cautious... This pattern is used elsewhere in the scripts.